bubblegum/program/src/lib.rs

```diff
@@ -92,11 +93,29 @@ pub struct Burn<'info> {
     pub gummyroll_program: Program<'info, Gummyroll>,
     /// CHECK: This account is checked in the instruction
     pub owner: UncheckedAccount<'info>,
-    /// CHECK: This account is chekced in the instruction
+    /// CHECK: This account is checked in the instruction
     pub delegate: UncheckedAccount<'info>,
-
     #[account(mut)]
+    /// CHECK: This account is modified in the downstream program
+    pub merkle_slab: UncheckedAccount<'info>,
+}
+
+#[derive(Accounts)]
+pub struct CreatorVerification<'info> {
+    #[account(
+        seeds = [merkle_slab.key().as_ref()],
+        bump,
+    )]
+    pub authority: Account<'info, TreeConfig>,
+    /// CHECK: This account is checked in the instruction
+    pub owner: UncheckedAccount<'info>,
     /// CHECK: This account is chekced in the instruction
+    pub delegate: UncheckedAccount<'info>,
+    pub creator: Signer<'info>,
```

Comment:
Wondering should I add a `pub payer: Signer<'info>,` account to these new Creator instructions.  In token-metadata we don't have a `payer` account for creator verification, but we do have one for all the Collection verification functions.